src/style.css

```diff
@@ -0,0 +1,385 @@
+  --light-color: #f4f4f4;
+  --red-color: rgb(216, 0, 0);
+  --green-color: #0f0;
+  --orange-color: #ffa500;
+  --pink-color: #ffc0cb;
+  --brown-color: #a52a2a;
+  --black-color: #000;
+  --gray-color: #808080;
+  --light-gray-color: #d3d3d3;
+  --dark-gray-color: #a9a9a9;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-family: 'Poppins', sans-serif;
+}
+
+li {
+  list-style: none;
+}
+
+h4 {
+  font-size: 13px;
+  font-weight: 800;
+  color: rgb(250, 17, 17);
+  background-color: rgba(0, 0, 0, 0.1);
+}
+
+span {
+  color: white;
+}
+
+body {
+  background-color: #000;
+  color: var(--primary-color);
+  overflow-x: hidden;
+  margin: 0;
+  padding: 0;
+  position: relative;
+}
+
+#hero-section {
+  color: var(--red-color);
+  width: 100vw;
+  height: 80vh;
+  background: linear-gradient(to bottom, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 1)), url('../dist/images/bg-for-page.jpg');
+  background-position: center;
+  background-size: cover;
+  background-repeat: no-repeat;
+  margin: 0;
+  padding: 0;
+  background-color: #000;
+}
+
+/* nav stylings */
+.nav-bar-home {
+  background-color: rgba(0, 0, 0, 0.5);
+}
+
+.navbar-brand {
+  font-size: 20px;
+  font-weight: 800;
+  color: var(--red-color);
+  text-shadow: #000 1px 1px 1px;
+}
+
+.nav-link {
+  color: var(--primary-color);
+  font-size: 14px;
+  font-weight: 800;
+  transition: all 0.4s ease-in-out;
+}
+
+.nav-link:hover,
+.navbar-brand:hover {
+  color: var(--red-color);
+}
+
+.welcomeTitle {
+  font-weight: 800;
+  font-size: 18px;
+  color: white;
+}
+
+.our {
+  color: var(--red-color);
+}
+
+.welcomeTitleWelc {
+  color: white;
+  font-size: 24px;
+  text-transform: capitalize;
+}
+
+.searchBtn {
+  background-color: var(--red-color);
+  border: none;
+  color: white;
+  font-weight: 800;
+  font-size: 14px;
+  transition: all 0.4s ease-in-out;
+  outline: var(--red-color);
+}
+
+/* cards stylings */
+.card {
+  border-radius: 10px;
+}
+
+.card img:hover {
+  box-shadow: 6px 6px 6px 0.5px rgba(255, 7, 7, 0.5);
+  transform: scale(1.06);
+  transition: all 0.3s ease-in-out;
+  background-color: white;
+}
+
+.custom-card {
+  background-color: transparent;
+  border: none;
+}
+
+.card-button {
+  display: flex;
+  flex-direction: column;
+  gap: 7px;
+  align-items: center;
+}
+
+.btn {
+  background-color: var(--red-color);
+  border: none;
+  transition: all 0.4s ease-in-out;
+  color: rgb(255, 255, 255);
+  font-weight: 600;
+  font-style: normal;
+}
+
+.btn:hover {
+  background-color: white;
+  color: var(--red-color);
+}
+
+.commentFormBtn {
+  color: white;
+  padding: 0.4rem;
+  border-radius: 5px;
+  background-color: red;
+}
+
+.commentFormBtn:hover {
+  color: red;
+  background-color: white;
+}
+
+.name-like-button {
+  display: flex;
+  justify-content: space-between;
+}
+
+/* modal styling */
+
+.modal {
+  width: 95vw;
+}
+
+.modal-body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
```

Comment:
In order to make the modals more visible, kindly add some back so that the text and content will be readable.

![image](https://github.com/Rafiduzzaman/Capstone-Project-Module2/assets/20554583/e2fa0404-749c-43d2-801f-20f62a3ddf64)